src/modules/governance/services/governance.abi.service.ts

```diff
@@ -383,6 +391,23 @@ export class GovernanceTokenSnapshotAbiService
             .buildTransaction()
             .toPlainObject();
     }
+
+    protected smoothing_function(quorum: string): string {
+        return new BigNumber(quorum).toFixed();
+    }
+
+    @ErrorLoggerAsync({ className: GovernanceTokenSnapshotAbiService.name })
+    @GetOrSetCache({
+        baseKey: 'governance',
+        remoteTtl: CacheTtlInfo.ContractState.remoteTtl,
+        localTtl: CacheTtlInfo.ContractState.localTtl,
+    })
+    async votingPowerDecimals(scAddress: string): Promise<number> {
```

Comment:
you can move this method in governance.service.ts

Reply:
i would keep it here so we can have caching on it

Reply:
done